repo-tools/tool-test-helper/generate

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+import os
+from datetime import datetime, timedelta
+
+import click
+
+INITIAL_PLUGIN_CONTENTS = """version: 0.1
+downloads:
+  - name: NAME_HERE
+    # executable: true
+    # NOTE: These are (common) sample values. Please replace with real values.
```

Comment:
Maybe mention something about how they're a mapping and can be resolved for ${version} ${os} ${cpu}